crypto/src/merkle.rs

```diff
@@ -212,8 +212,8 @@ impl<T> MerkleTree<T> {
 
     #[cfg(feature = "std")]
     fn nodes_pair_hash(
-        l_node: &Option<HashOf<T>>,
-        r_node: &Option<HashOf<T>>,
+        l_node: Option<&HashOf<T>>,
+        r_node: Option<&HashOf<T>>,
```

Comment:
Please, also take a look at **line 75** in this file. It also contains `&Option<...>` instead of `Option<&...>`

Reply:
It breaks the trait implementation of `CompleteBTree<T>::get()` if we use `Option<Option<&...>>` instead of `Option<&Option<...>>`.
So I do not think it is possible to change it.